roles/scale_hdfs/upgrade/tasks/zypper/upgrade.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: upgrade | Upgrade GPFS HDFS packages
+  zypper:
+   name: "{{ scale_upgrade_all_packages }}"
+   state: latest
+   disable_gpg_check: no
+  when: ansible_fqdn in scale_hdfs_node_list
```

Comment:
I think, we may need to remove this when condition.

Reply:
done